packages/perseus-editor/src/item-extras-editor.tsx

```diff
@@ -72,6 +79,13 @@ class ItemExtrasEditor extends React.Component<Props> {
                                     this.props.onChange({
                                         periodicTable: e.target.checked,
                                     });
+                                    if (!e.target.checked) {
+                                        // When we remove the periodic table,
+                                        // we should also remove sub-options.
+                                        this.props.onChange({
+                                            periodicTableWithKey: false,
+                                        });
+                                    }
```

Comment:
**thought**: We could put these within the same `onChange` callback. That'd save one extra re-render. But maybe I shouldn't be thinking at that level and just let it re-render. :) You're call.

Reply:
Yeah, I can do that. I am just not used to class component patterns.